glide.yaml

```diff
@@ -4,8 +4,16 @@ licence: Apache-2.0
 
 import:
 
+- package: github.com/sirupsen/logrus
+  version: 1.0.3
+
 - package: github.com/Sirupsen/logrus
-  version: v0.11.5
+  repo: git@github.com:/sirupsen/logrus
```

Comment:
Is there another way to resolve this other than vendoring this twice?

Reply:
Nope :-1: https://github.com/sirupsen/logrus/issues/570